static/docs/use-cases/data-registries.md

```diff
@@ -42,6 +38,10 @@ Advantages of using a DVC **data registry** project:
 - Security: Registries can be setup to have read-only remote storage (e.g. an
   HTTP location). Git versioning of [DVC-files](/doc/user-guide/dvc-file-format)
   allows us to track and audit data changes.
+- Data as code: Improve _lifecycle management_ with versioning of simple
+  directory structures (like Git on cloud storage), without ad-hoc conventions.
+  Leverage Git and Git hosting features such as commits, branching, pull
+  requests, reviews, and even continuous deployment of ML models.
```

Comment:
@shcheklein you wanted to rephrase this AND move it to the bottom, or should we return it to the top but with simpler wording?

Reply:
I think it's fine to have it bottom, but we should simplify and clarify it

Reply:
ApplieD

> Leverage Git workflow such as commits, branching, pull requests, reviews, and even continuous deployment for your data and models lifecycle.

From https://github.com/iterative/dvc.org/pull/869#pullrequestreview-332886790